settings.py

```diff
@@ -1369,8 +1367,6 @@
     ('app_manager', APPS_DB),
 ]
 
-COUCHDB_APPS += LOCAL_COUCHDB_APPS
```

Comment:
I think this was meant to add `LOCAL_COUCHDB_APPS` from `localsettings.py`

https://github.com/dimagi/commcare-hq/blob/e30e71b1130c026f3251a5adc86119556d7cea2a/settings.py#L888

to `COUCHDB_APPS`. How did you verify that no one has set `LOCAL_COUCHDB_APPS` in their localsettings.py?

Reply:
Seemed unlikely that people had ```LOCAL_COUCHDB_APPS``` or ```LOCAL_REPEATERS``` set since these variables aren't even in ```localsettings.example.py```.